docs/asciidoc/uuid.adoc

```diff
@@ -0,0 +1,110 @@
+== UUID
+
+There is the possibility to set uuid index property on nodes using APOC.
+
+Enable `apoc.uuid.enabled=true` in `$NEO4J_HOME/config/neo4j.conf` first.
+
+[cols="1m,5"]
+|===
+| CALL apoc.uuid.add(label, config) yield label, installed | add a uuid statement returns previous and new uuid information
+| CALL apoc.uuid.remove(label) yield label, installed | remove previously added uuid, returns uuid information
+| CALL apoc.uuid.removeAll() yield label, installed | removes all previously added uuids, returns uuid information
+| CALL apoc.uuid.list() yield label, installed | get all the uuid installed
+|===
+
+==== Config
+
+|===
+|config | default | description
+|addToAlreadyPresent | true | the uuid for existing node
```

Comment:
can we find a better name, it is not really clear what this means and when it is important.

also better docs

Reply:
ok